docs/rules/yield-star-spacing.md

```diff
@@ -1,41 +1,39 @@
-# Enforce spacing around the `*` in `yield*` expressions (yield-star-spacing)
+# Enforce spacing around the `*` in `yield*` statements (yield-star-spacing)
```

Comment:
`yield` is not statements. It's expressions.

http://www.ecma-international.org/ecma-262/7.0/#prod-YieldExpression